apps/cowswap-frontend/src/legacy/components/Header/styled.tsx

```diff
@@ -232,7 +209,7 @@ export const Wrapper = styled.div<{ isMobileMenuOpen: boolean }>`
       ${
         isMobileMenuOpen &&
         css`
-          position: fixed;
+          position: absolute;
```

Comment:
Should this stay?

Reply:
I have no clue. I didn't notice any change. @fairlighteth WDYT? can you follow up if this is a change that should be done?